### DISCUSSION TITLE
Filtered Views Not Saving

### DISCUSSION THREAD

**Discussion by liserubarth** [community/community#7652] [Projects and Issues]
In GitHub Beta, I create a view, add a filter, then save.
I come back to the tab and the view is there, but not the filter.
I would like to be able to save filtered views as part of the saved tab view.

**Comment by azenMatt** [comment]
Hi @liserubarth - are you still having problems with this? Anything mutation you add and then save should persist. 

Do you mind sharing a video of this to help us debug it?

**Comment by liserubarth** [comment]
Hello, thanks for contacting me. This issue seems to be resolved now,
thanks!
